gen/enum.go

```diff
@@ -26,6 +26,11 @@ import (
 	"go.uber.org/thriftrw/compile"
 )
 
+const (
+	// GoWireLabel overrides name on the wire
+	GoWireLabel = "go.wire.label"
```

Comment:
Let’s call this just `"label"` or `"go.label"` since it affects the JSON and log representation of the in memory struct and not the TBinary (wire) protocol.